config/locales/en.yml

```diff
@@ -263,22 +263,22 @@ en:
         address_telephone: Phone number (optional)
       events_steps_personalised_updates:
         address_postcode: What is your postcode? (optional)
-        degree_status_id: What stage are you at with your degree?
-        consideration_journey_stage_id: How close are you to applying for teacher training?
+        degree_status_id: Do you have a degree?
+        consideration_journey_stage_id: What stage are you at in applying?
```

Comment:
Only question I'm unsure on. Yo me, it implies that they've already begun applying. Did you consider any other variants?

Reply:
I think we can stick to 'How close are you to applying for teacher training?' - was just playing around with this one.